cv_next/lib/definitions.ts

```diff
@@ -8,13 +8,44 @@ export default class Definitions {
 }
 
 export const heroHeader: PageHeader = {
-  header: `Landing pages made easy`,
-  subheader: `Easy to setup. Customizable. Quick. Responsive.`,
-  image: `/hero-img.webp`,
+  header: `Next`,
+  image: `/images/logo.png`,
+  subheader: `improve the magshimim next community's resumes.`,
+  explenetion: `we as a magshimim next community can help each other get jobs by improving our memeber's resumes.
+  this website is a place to uplode resumes, let other review them and make them better.
+  we will not tolerate any abuse, hateful commects, griffing, pentesting and etc... `
```

Comment:
Improve the Magshimim Next Community's resumes.

As a community, we can help each other secure jobs by enhancing our members' resumes. This website is a platform to upload resumes, allow others to review them, and make them better.
We expect all interactions to be respectful and constructive, with no tolerance for abuse, hateful comments, or any form of inappropriate behavior.